### DISCUSSION TITLE
Tables Style in the Documentation

### DISCUSSION THREAD

**Discussion by AbelMH1** [Arquisoft/wiq_es6c#93] [Polls]
While revising the documentation for our app I have seen many different styles for the tables in it. I do not have any preference for any of them, so I was hoping you could help me to choose.
Here are the different styles applied to the same table (they only differ on the first column):

**Comment by marco-qg** [comment]
Option 2 🎩

**Comment by AbelMH1** [comment]
It has been decided by majority vote that the option to be used will be **option 2** (italics only)
